### DISCUSSION TITLE
Llama 3 is out

### DISCUSSION THREAD

**Discussion by Galunid** [ggml-org/llama.cpp#6746] [General]
https://ai.meta.com/blog/meta-llama-3/
https://llama.meta.com/llama-downloads

**Comment by mirek190** [comment]
..and is insane good ...

**Comment by stewartoallen** [comment]
Even building llama.cpp from today's source with the Llama 3 patches, when using most Llama 3 models (with `main`), it's still wandering off into gibberish land after generating a few hundred tokens. Using the same prompt and same Llama 3 models in something like LM Studio (which uses llama.cpp), I get coherent results. What am I missing?
